chanzuckerberg/idseq-web
# Add new info for nonhost fastq step

# Description

This goes with https://github.com/chanzuckerberg/idseq-dag/pull/282 . 

# Tests

* run  bundle exec 'rake export_dags[15149,1]'
* see `use_taxon_whitelist` and both cdhitdup files
* tested locally with branch of https://github.com/chanzuckerberg/idseq-dag/pull/282 .
